### ISSUE TITLE
Proxy option not working when websql adapter used for local db

### ISSUE THREAD

**Issue by pove** [nolanlawson/pouchdb-load#35]
When your local db is created with this constructor:
```javascript
new PouchDB('idstoneDB', { adapter: 'websql' });
```
the handoff to regular replication with proxy option does not work. This is becouse it tries to open proxy database with websql option as well, instead of use http o whatever.

This is my code:
```javascript
var series = PouchDB.utils.Promise.resolve();

dumpFiles.forEach(function (dumpFile) {
   series = series.then(function () {
	   return db.load(dumpFile,
	   {
		   proxy: 'http://192.168.0.101:4984/sync_db',
		   auth: {
			   username: 'userone',
			   password: 'passwd'
		   }
	   }
   ).then(function (data) {
	   // done!
   }).catch(function (err) {
	   // HTTP error or something like that
   });
   });
});

series.then(function () {
   // done loading!
   return db.put({ _id: '_local/initial_load_complete' });
}).catch(function (err) {
   // HTTP error or something like that
});
```

I've tryed also forcing adapter to http, but it changes also the local db adapter option:
```javascript
[...]
series = series.then(function () {
	   return db.load(dumpFile,
	   {
		   proxy: 'http://192.168.0.101:4984/sync_db',
		   auth: {
			   username: 'userone',
			   password: 'passwd'
		   },
		   adapter: 'http'
	   }
[...]
```

**Comment by pove** [comment]
It works if you make the next change on loadString function:

```javascript
var src = new db.constructor(opts.proxy,
      //utils.extend(true, {}, db.__opts, opts)); !!! replace this line for the next one:
      utils.extend(true, {}, { adapter: 'http' }, opts));
var target = new db.constructor(info.db_name,
      utils.extend(true, {}, db.__opts, opts));

**Comment by nolanlawson** [comment]
fixed by 2edb5f6652fc171cc7935fbfd6324e913cca2835
